hack/ci/e2e-conformance.sh

```diff
@@ -69,6 +71,34 @@ dump-logs() {
 
   # export all logs from kind
   kind "export" logs --name="clusterapi" "${ARTIFACTS}/logs" || true
+
+  jump_node=$(aws ec2 describe-instances --region $AWS_REGION --query "Reservations[*].Instances[*].PublicIpAddress" --output text | head -1)
+  for node in $(aws ec2 describe-instances --region $AWS_REGION --query "Reservations[*].Instances[*].PrivateIpAddress" --output text | tail -n +2)
+  do
+    echo "collecting logs from ${node} using jump host ${jump_node}"
+    dir="${ARTIFACTS}/logs/${node}"
+    mkdir -p ${dir}
+    ssh-to-node "${node}" "${jump_node}" "sudo journalctl --output=short-precise -k" > "${dir}/kern.log" || true
+    ssh-to-node "${node}" "${jump_node}" "sudo journalctl --output=short-precise" > "${dir}/systemd.log" || true
+    ssh-to-node "${node}" "${jump_node}" "sudo crictl version && sudo crictl info" > "${dir}/containerd.info" || true
+    ssh-to-node "${node}" "${jump_node}" "sudo journalctl --no-pager -u cloud-final" > "${dir}/cloud-final.log" || true
+    ssh-to-node "${node}" "${jump_node}" "sudo journalctl --no-pager -u kubelet.service" > "${dir}/kubelet.log" || true
+    ssh-to-node "${node}" "${jump_node}" "sudo journalctl --no-pager -u containerd.service" > "${dir}/containerd.log" || true
+  done
+}
+
+# SSH to a node by name ($1) via jump server ($2) and run a command ($3).
+function ssh-to-node() {
+  local node="$1"
+  local jump="$2"
+  local cmd="$3"
+
+  ssh_key_pem="/tmp/${SSH_KEY_NAME}.pem"
+  scp -i $ssh_key_pem $ssh_key_pem "ubuntu@${jump}:$ssh_key_pem"
+  ssh_params="-o LogLevel=quiet -o ConnectTimeout=30 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
+  ssh $ssh_params -i $ssh_key_pem \
+    -o "ProxyCommand ssh $ssh_params -W %h:%p -i $ssh_key_pem ubuntu@${jump}" \
```

Comment:
Can you use `ssh -i $ssh_key_pem -J ubuntu@${jump} ubuntu@${node}` instead? I think you can avoid having to copy the pem over, or if not, you could use a temporary `ssh-agent` to do it.

Reply:
the `-J ubuntu@${jump} ubuntu@${node}` didn't work out of the box. this worked when i tried by hand. will have to look into `ssh-agent` as well.

Reply:
Sounds good. I use `-J` all the time, albeit with my key loaded into `ssh-agent`.

Reply:
right, i need to find a balance since this script can be run by a developer as well. this path of creating a new key and wiping it off seemed a better approach.

Reply:
Would it be worth it to leverage the aws systems manager run command instead? https://docs.aws.amazon.com/systems-manager/latest/userguide/execute-remote-commands.html

That said, I'm good with that as a followup/longerterm approach.

Reply:
@detiber yes, we could explore other options